gtsam/discrete/DiscreteFactor.h

```diff
@@ -104,6 +94,6 @@ class GTSAM_EXPORT DiscreteFactor: public Factor {
 
 // traits
 template<> struct traits<DiscreteFactor> : public Testable<DiscreteFactor> {};
-template<> struct traits<DiscreteFactor::Values> : public Testable<DiscreteFactor::Values> {};
+template<> struct traits<DiscreteValues> : public Testable<DiscreteValues> {};
```

Comment:
We should move this to `DiscreteValues.h`.

Reply:
Good catch!!!